app/uk/gov/hmrc/tradergoodsprofiles/models/errors/ErrorResponse.scala

```diff
@@ -87,6 +87,7 @@ case class ServerErrorResponse(correlationId: String, message: String) {
     )
 }
 
+//todo: see if we can remove this
 case class BadRequestErrorResponse(correlationId: String, code: String, message: String, errorNumber: Int) {
```

Comment:
can we remove this, please. I think is not used any where anymore

Reply:
Done ✅